tests/py/test_email.py

```diff
@@ -219,6 +220,13 @@ def test_html_escaping(self):
         assert 'foo&#39;bar' in last_email['html']
         assert '&#39;' not in last_email['text']
 
+    @mock.patch('gratipay.models.participant.Participant.send_email')
+    def test_emails_with_plus_are_linked_properly(self, send_email):
+        send_email.return_value = 1
+        self.alice.add_email("foo+bar@example.com")
+        link = send_email.args[1]
+        assert link.startswith('/~alice/emails/verify.html?email=foo%2Bbar%40example.com&nonce=')
+
```

Comment:
I modified the verification link to contain `email64=<encoded-email>`. Here's the [test](https://github.com/gratipay/gratipay.com/commit/5f9320c4ec20a5bd1d88ed7072e363e60d0c0223#diff-a4f4fd2e3f2cfd01d60403c0c4c3a02fR54)